contracts/SwapAndBridge.sol

```diff
@@ -79,11 +86,13 @@ abstract contract SwapAndBridgeBase is Lockable, MultiCaller {
      */
     constructor(
         V3SpokePoolInterface _spokePool,
+        WETH9Interface _wrappedNativeToken,
```

Comment:
@bmzig This is also able to be queried directly from the SpokePool contract. Thoughts on doing that within the constructor instead? It'd remove a constructor argument and would also eliminate the possibility for an (admittedly unlikely) misalignment on the address.

Reply:
I thought of doing that but `wrappedNativeTokens` isn't exposed by the V3SpokePoolInterface/SpokePoolInterface, so we would need to modify the interface for this to work (or just create a new interface). I figured we probably shouldn't do that.